contrast-ratio.js

```diff
@@ -6,8 +6,12 @@ function $$(expr, con) {
 	return Array.prototype.slice.call((con || document).querySelectorAll(expr));
 }
 
-// Make each ID a global variable
-// Many browsers do this anyway (it’s in the HTML5 spec), so it ensures consistency
+/*
+ * Make each element with an ID a global variable.
+ * Many browsers do this anyway (it’s in the HTML5 spec), so it ensures consistency.
+ *
+ * https://html.spec.whatwg.org/multipage/window-object.html#named-access-on-the-window-object
```

Comment:
This reminds me, at this point is there any current browser that *doesn't* do this? If not, we could just remove this code.

Reply:
This comment actually taught me that this was a thing at all. I had no idea this was standard behavior. When I looked into it, I mostly found comments to avoid depending on that part of the spec, though. (i.e. if in future there is a future global called `background`, then it'll break the site) ^-^' 

I can only confirm that Chromium and Firefox Nightly have this behavior by default, though.

Reply:
Yeah, we should probably keep the loop but assign them to some custom object (e.g. `dom`).